lib/email_updater.rb

```diff
@@ -11,18 +11,10 @@ def initialize(guardian: nil, user: nil, initiating_user: nil)
     @initiating_user = initiating_user
```

Comment:
@martin-brennan Do you recall why we have to pass in the `initiating_user` which can be `nil`? It doesn't seems right to me because the initiating use can always be retrieved via `guardian` and `EmailUpdater` does not allow any anonymous actions.

Reply:
@tgxworld guess just a brain-fart on my part, I don't see any reason to keep `initiating_user` here if we can just do `guardian.user`

Reply:
@martin-brennan Cool :+1: Let me see if I can remove it while not breaking any tests :P